spack_environments/std_versions_and_variants_llnl_lc_cz.yaml

```diff
@@ -43,3 +43,47 @@
     zlib::
       buildable: True
       version: [1.2.11]
+
+  modules::
+    enable::
+      - tcl
+      - lmod
+    lmod::
+      hash_length: 3
+      core_compilers:
+        - 'gcc@4.9.3'
```

Comment:
Should you add `gcc@7.3.0` since we actually use that one?

Reply:
Done.